src/cca/app/scripts/cordova_browser_api.ts

```diff
@@ -343,6 +347,14 @@ class CordovaBrowserApi implements BrowserAPI {
     }
   }
 
+  public exit = () => {
+    (<any>navigator).Backbutton.goBack(() => { /* MT (success) */ }, () => {
```

Comment:
Nit: I think it would be worthwhile to add a minimal d.ts file like https://github.com/uProxy/uproxy/blob/master/third_party/cordova/splashscreen.d.ts